src/main/java/net/comments/ft/Test5.java

```diff
@@ -0,0 +1,29 @@
+package net.comments.ft;
+
+import net.comments.objects.BCommentPage;
+import net.comments.objects.Comment;
+import net.comments.objects.CommentTable;
+import org.hamcrest.MatcherAssert;
+import org.testng.annotations.Test;
+
+public class Test5 extends BaseDriver {
+    private final BCommentPage page;
+
+    Test5() {
+        this.page = new BCommentPage(driver);
+    }
+
+    @Test
+    public void test() {
+        page.open();
+        CommentTable commentTable = page.getCurrentComments();
+        int[] commentNumbers = {2, 5};
+        page.getCurrentComments().selectComment(commentNumbers);
```

Comment:
Why do you use an array here? If you need to select 2 comments, use method twice.